code/modules/client/verbs/ooc.dm

```diff
@@ -317,8 +317,8 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 
 	var/message = "<span class='big'>You can add emphasis to your text by surrounding words or sentences in certain characters.</span>\n \
 		**bold**, and _italics_ are supported.\n\n \
-		<span class='big'>You can made custom saymods by doing <i>say 'screams- HELP IM DYING!'</i>. This works over the radio, and can be used to emote over the radio.</span>\n \
-		Example: say ';laughs maniacally!-' >> \[Common] Joe Schmoe laughs maniacally!"
+		<span class='big'>You can made custom saymods by doing <i>say 'screams\\\ \\ HELP IM DYING!'</i>. This works over the radio, and can be used to emote over the radio.</span>\n \
```

Comment:
this should be one backslash, so just one escaped backslash set: `\\`

Reply:
Made it that many backslashes in order to make it show up as `\\` in the OOC help thing, maybe there's some other way to make it only render two of them? This is to hopefully prevent people from accidentally ending up doing the thing I talked about above.